docs/resources/default_namespace_settings.md

```diff
@@ -0,0 +1,26 @@
+---
+subcategory: "Settings"
+---
+
+# databricks_namespace_settings Resource
+
```

Comment:
I chatted with @mgyucht on the doc part. Let's put a note here saying that this API is private preview mode currently. We will remove that "private preview mode" note and update this doc to include proper documentation after we enable the UC setting public API in production. @hectorcast-db

Reply:
Previously we just released PrPr functionality without public documentation, but included TF docs into PrPr user guide. I think that we need to follow that pattern here as well.

Cc: @mgyucht

Reply:
I see, thanks for the context @alexott. Happy to follow the existing process, I didn't know what the status quo was here.

Reply:
Added the documentation shared by slack.